python/indy-agent.py

```diff
@@ -49,7 +49,6 @@
     web.get('/ws', AGENT['ui_event_queue'].ws_handler),
     web.static('/res', 'view/res'),
     web.post('/indy', AGENT['msg_receiver'].handle_message),
-    web.post('/offer', connection.offer_recv)
```

Comment:
How are connections offers sent if this handler is removed? The intention in keeping this and the `/indy` enpoint separate was so that the distinction between how "in-band" Sovrin messages are received and processed and the "out-of-band" connection offers are received and processed. The reasoning here is very much steeped in discussion that's been going on around how agents operate in the Sovrin world and we hope to have aHIPE (Hyperledger Indy Project Enhancement) available soon that outlines this connection process.

So, if possible, I think it would be better to process the connection offers separately.